app/assets/javascripts/application.js

```diff
@@ -129,6 +130,25 @@ var paginationOpts = {
     position: 'relative' // Element positioning
 };
 
+// global config for CKEditor instances
```

Comment:
I know this file is inconsistent throughout, but at least for new stuff, let's settle on 2-spaces indent for JS

Reply:
(Seconded.)  If not done already, I think this might be best done by integrating [this repo's `.eslintrc.js`](https://github.com/broadinstitute/single_cell_portal_core/blob/master/.eslintrc.js#L58) into your IDE's settings.  That ought to fix all JS style issues.  If the `eslintrc.js` file doesn't catch this spacing issue, we should amend it.